cli/user/src/main/java/io/pravega/cli/user/config/InteractiveConfig.java

```diff
@@ -28,6 +29,7 @@
 @Data
 @Builder
 public class InteractiveConfig {
+    public static final String LOGLEVEL = "log-level";
```

Comment:
minor: `LOG_LEVEL` follows the naming convention for constants here.
```suggestion
    public static final String LOG_LEVEL = "log-level";
```